libeos-payg/efi.c

```diff
@@ -604,42 +604,6 @@ eospayg_efi_secureboot_setup_active (void)
   return flag ? EFIVAR_TRUE : EFIVAR_FALSE;
 }
 
-/* eospayg_efi_securebootoption_disabled:
- *
- * Check if the SecureBootOption EFI variable exists and
- * is disabled.
- *
- * The oddly inverted logic is due to the fact that most
- * systems don't have this variable at all - if it doesn't
- * exist, we can infer nothing about the state of Secure Boot,
- * if it does it tells us if the Secure Boot option in the
- * BIOS is enabled or disabled.
- *
- * Thus, on and not existing should likely be treated in the
- * same way by a caller, but existing and off is a red flag
- * for PAYG enforcement.
- *
- * Returns: %TRUE if the variable exists and is disabled, %FALSE otherwise
- */
-gboolean
-eospayg_efi_securebootoption_disabled (void)
-{
-  g_return_val_if_fail (efi != NULL, FALSE);
-
-  g_autofree char *tname = full_efi_name (SBO_VARIABLE_GUID, "SecureBootOption");
```

Comment:
I think `SBO_VARIABLE_GUID` is now unused and can be removed.

Reply:
Good point!  It becomes an orphan